src/main/java/javaClasses/homework_4/Lilit_Manukyan/Calculations.java

```diff
@@ -0,0 +1,41 @@
+package javaClasses.homework_4.Lilit_Manukyan;
+
+public class Calculations {
+    public static int getMean(int... args) {
+        int sum = 0;
+        int count = 0;
+        for (int el : args) {
+            count += 1;
+            sum += el;
+        }
+        return sum / count;
```

Comment:
Nice job! But keep in mind that args is actually treated like an array. So it would be totally ok to use args.length instead of keeping separate count variable. Please change it just for 1 method, to see how it works.

Reply:
Done